### PR TITLE
adding new fields and relationships

### PR DESCRIPTION
**Pull request by amatosg**

### CODE REVIEW

File: bug-tracker.jh [hunk 1]
```diff
@@ -20,6 +28,44 @@ relationship ManyToMany {
...
+
+enum Type {
+  BUG("Bug"),
+  FEATURE("Feature")
+}
+
+enum Priority {
+  HIGHEST("Highest")
+  HIGHER("Higher")
+  HI("Hi")
```
Comment (atomfrede) [inline bug-tracker.jh]: I think it should be `HIGH("HIGH")`

```suggestion
HIGH("HIGH")
```

**Review by atomfrede — CHANGES_REQUESTED** [review]
Besides the small typo it looks good to me.

File: bug-tracker.jh (hunk 1 above)
Comment (amatosg) [reply bug-tracker.jh]: thanks! typo fixed

**Comment by amatosg** [comment]
@atomfrede can this be merged now?

**Review by atomfrede — APPROVED** [review]

**Comment by atomfrede** [comment]
@amatosg Sorry, totally forgot about this. Let's merge it. Thanks!

**Pull request merged**